src/blinky/constr/nextpnr/xc7_a35t_csg324-1_arty.xdc

```diff
@@ -0,0 +1,51 @@
+# R
```

Comment:
Shouldn't the constraints be the same for nextpnr/vivado/vpr?

Reply:
So, vivado and nextpnr should be able to share the xdc constraints. Cannot be the same for the vpr symbiflow toolchain.

I think that a work to be done is to be able to express all the constraints (clock, IOs) in the xdc format for the symbiflow toolchain as well, so that we can have a unified constraint file for all the toolchains.

As it is currently, the tests are being run by the exhaust based on the presence of a mandatory constraint file in the srcs directory.

Reply:
Please log an issue about making everyone use the same constraint file.